server/src/main/java/com/thoughtworks/go/server/messaging/PluginMessageQueueHandler.java

```diff
@@ -69,9 +73,12 @@ public void post(T message, long timeToLive) {
         String pluginId = message.pluginId();
```

Comment:
We still need to think through about how to handle heavy load, I intentionally set the ttl to 0 for notification plugin since I dont know what the repercussion of expiring these messages would mean, but I guess this is something that needs to be revisited, hence I am not in favour of  overloading the method just yet.

Reply:
ok